pkg/cmd/start/start.go

```diff
@@ -125,8 +124,7 @@ func runGather(operator *controller.GatherJob, cfg *controllercmd.ControllerComm
 // Boilerplate for running an operator and handling command line arguments.
 func runOperator(operator *controller.Operator, cfg *controllercmd.ControllerCommandConfig) func(cmd *cobra.Command, args []string) {
 	return func(cmd *cobra.Command, args []string) {
-		// boiler plate for the "normal" command
-		rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
https://go-review.googlesource.com/c/go/+/443058

Reply:
Hmm I wonder why the random generator was seeded....but hopefully it can be removed since in Go 1.20 it should be auto-seeded

Reply:
Exactly, the `< 1.20` we had to seed the random, from `>= 1.20` it isn't necessary anymore.